distributed/utils.py

```diff
@@ -1451,7 +1453,7 @@ def import_term(name: str):
 
     Examples
     --------
-    >>> import_term("math.sin")
+    >>> import_term("math.sin") # doctest: +SKIP
```

Comment:
I chose to skip here because the actual result was `<built-in function sin>`. But we could always go the other way.

Reply:
I'm fine either way. I personally would probably only skip if it cannot be executed for some reason but I don't think this is really crucial